### DISCUSSION TITLE
Here is my rendition of the Squishbox

### DISCUSSION THREAD

**Discussion by clendee** [GeekFunkLabs/fluidpatcher#29] [Show and tell]
Thanks for your comments and your willingness to explore the questions and ideas.

Now that I have built the Squishbox, I can get more stuck in to its capabilities.  Here is my rendition of it.  

A friend of mine has a laser cutter and built the box out of 3mm board.  Its quite study and fits neatly.  He said he'd happily share his design for any interested parties.

![IMG_20211122_165239](https://user-images.githubusercontent.com/50013459/142929314-f407059d-6751-4bd4-acfe-a2b94ba795d9.jpg)

**Comment by albedozero** [comment]
Nice! I might need to come up with a good landing place for folks to share their creations/content related to this. Maybe the [Hackaday project page](https://hackaday.io/project/9097-squishbox#j-discussions-title)? Discord? Facebook (blerg)?

**Reply by clendee** [reply]
Is there a reason you don't want to drive people to the Github Discussions board, like we are using.  I'm afraid I am not on Facebook or any platforms provided my the major players.  I guess it depends.  A lot of content producers have a discord server \ channel for people to chat (instant messenger style) about their ideas etc...  I guess github discussions dont really keep that flow going.  I'm more than happy to follow your lead on the decision.  [except I won't join FB ].

Perhaps a chat platform might be good, people could jump around with it.  I dunno.  A telegram channel?

**Reply by albedozero** [reply]
I was just thinking GitHub Discussions are great too - you can attach almost any files you want (they can be tucked into a .zip if github rejects a certain file type b/c security reasons). I do haz [Discord](https://discord.gg/Qs4gmHf) if people want to use it, but it's a bit desolate at the moment 😥. I'll engage with people wherever they feel comfortable - just curious what's more convenient or what platforms are out there I haven't heard of.

**Reply by clendee** [reply]
Ok, I see.  Yeah, I understand.  I've pushed your project around in my channels.  I used to run a podcast called '[The Binary Times](https://thebinarytimes.net/)', [its stopped now], but we have a few audio interested parties in the remnants of the group.  We have a Matrix channel for our discussions, but I understand thats a bit niche.  https://matrix.org/  There are loads of musicians out there, and quite a few geeks.  It can be good not to spread your digital life too thin, so maybe hang out in github discussions for a while and keep content coming in there.  I'm happy to continue feeding back, if you're up for it.
